src/api/privateApi.js

```diff
@@ -67,61 +67,61 @@ export function getURLPrefix() {
 
 export function getFullURL() {
 	const nodeUrl = this.port
-		? `${this.currentPeer}:${this.port}`
-		: this.currentPeer;
+		? `${this.node}:${this.port}`
+		: this.node;
 
 	return `${getURLPrefix.call(this)}://${nodeUrl}`;
 }
 
 /**
- * @method getPeers
- * @return peers Array
+ * @method getNodes
+ * @return nodes Array
  * @private
  */
 
-export function getPeers() {
-	if (this.testnet) return this.defaultTestnetPeers;
-	if (this.ssl) return this.defaultSSLPeers;
-	return this.defaultPeers;
+export function getNodes() {
+	if (this.testnet) return this.defaultTestnetNodes;
+	if (this.ssl) return this.defaultSSLNodes;
+	return this.defaultNodes;
 }
 
 /**
- * @method getRandomPeer
- * @return peer string
+ * @method getRandomNode
+ * @return  string
  * @private
  */
 
-export function getRandomPeer() {
-	const peers = getPeers.call(this)
-		.filter(peer => !this.bannedPeers.includes(peer));
+export function getRandomNode() {
+	const nodes = getNodes.call(this)
+		.filter(node => !this.bannedNodes.includes(node));
 
-	if (!peers.length) {
-		throw new Error('Cannot get random peer: all relevant peers have been banned.');
+	if (!nodes.length) {
+		throw new Error('Cannot get random node: all relevant nodes have been banned.');
 	}
 
-	const randomIndex = Math.floor((Math.random() * peers.length));
-	return peers[randomIndex];
+	const randomIndex = Math.floor((Math.random() * nodes.length));
+	return nodes[randomIndex];
 }
 
 /**
  * @method selectNode
```

Comment:
Can we give this function a more illuminating name? `setNode`, `refreshNode`, `selectNewNode`?